config/jobs/kubernetes/sig-node/sig-node-presubmit.yaml

```
@@ -1224,7 +1224,7 @@ presubmits:
     always_run: false
     optional: true
     skip_report: false
```

```suggestion
    skip_report: true
```

you can also skip reporting if you want, so it runs but is not surfaced to the users

---

sounds good, updated as you suggested.